impeller/renderer/backend/vulkan/swapchain_image_vk.h

```diff
@@ -30,12 +30,19 @@ class SwapchainImageVK final : public TextureSourceVK {
   // |TextureSourceVK|
   vk::Image GetImage() const override;
 
+  std::shared_ptr<Texture> GetMSAATexture() const;
+
+  bool HasMSAATexture() const;
+
   // |TextureSourceVK|
   vk::ImageView GetImageView() const override;
 
+  void SetMSAATexture(std::shared_ptr<Texture> msaa_tex);
+
  private:
   vk::Image image_ = VK_NULL_HANDLE;
   vk::UniqueImageView image_view_ = {};
+  std::shared_ptr<Texture> msaa_tex_;
```

Comment:
This is rough as these textures are huge. We should measure memory usage but I doubt we can use the Vulkan backend in this instance. This is just the the swapchain images. These will add up for intermediates as well.

If this is to stabilize the benchmarks, its fine I suppose. But we should then find a device to put in the lab that does support lazily allocated images.

Reply:
Yeah, I'd like to punt on whether or not we decided to ship Vulkan on these devices. We'll likely learn even more once we actually attempt the Vulkan to GL texture interop.

Though for the short term, we must keep something on CI using Vulkan and I don't think we have any other devices.

Reply:
>  I doubt we can use the Vulkan backend in this instance

Couldn't we use a different AA then MSAA, like fxaa?